diff --git a/conf/config.yaml b/conf/config.yaml
index dcea755..809fca0 100644
--- a/conf/config.yaml
+++ b/conf/config.yaml
@@ -2,4 +2,5 @@ defaults:
   - params
 
 experiment_name: "test_experiment"
+new: "test with modified test script"
 experiment_metadata_filepath: "experiment_metadata.txt"
\ No newline at end of file
diff --git a/scripts/test.sh b/scripts/test.sh
index 3978d01..1ca7c97 100755
--- a/scripts/test.sh
+++ b/scripts/test.sh
@@ -51,7 +51,7 @@ function cleanup_job_dir(){
     git stash pop
 
     # make a submission log
-    JOB_ID=$(RANDOM)
+    JOB_ID=$(echo $RANDOM)
     echo "$(date %Y/%m/%d %H:%M:%SS),${EXPERIMENT_NAME},${JOB_ID}" >> $JOB_LOGS
     echo "Job submitted!"
 }
